### ISSUE TITLE
Anybody get the paper's result? Rank1 90%.

### ISSUE THREAD

**Issue by AsuradaYuci** [jackie840129/STE-NVAN#5]
As I don't have enough GPU memorys, So I can't run the Non-local version code. But I am curious about the Non-local performance.

**Comment by jackie840129** [comment]
Hi @AsuradaYuci ,
What kind of your GPU?
If you have about two Titan xp (about 24GB memory), you can train with 3 non-local layers.(the argument is --non_layers 0 1 2 0 ). And you can get approximately 88~89% rank-1 acc.

**Comment by AsuradaYuci** [comment]
@jackie840129 ，Thanks for your suggestion, but I only have a GPU....

**Comment by jackie840129** [comment]
Hi @AsuradaYuci ,
Because the task is video-based re-id, it truly needs lots of GPU memory...sorry.
I think other papers without sampling the input sequence will need more lol.
The only work maybe can run on one GPU is the STA (https://arxiv.org/abs/1811.04129).
This work didn't provide the source code, maybe you can try reproduce it~ ><.

**Comment by AsuradaYuci** [comment]
Hello @jackie840129 ,
Do you think  the hyperparameters **BatchSize** plays a important role in performance? Maybe the  batchsize is larger, the performance will be better.  And thanks for your suggestion

**Comment by jackie840129** [comment]
Hi @AsuradaYuci ,

The performance also depends on batchsize.
My batch size depends on the argument " track_per_class(8)" and "class per batch(4)" and "S(8)".
The number must be large enough to increase the diversity to form the positive and negative pairs in order to train the triplet loss.
